scripts/nbconverted/generate_input_PA1673like_v1.py

```diff
@@ -0,0 +1,169 @@
+
+# coding: utf-8
+
+# In[1]:
+
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# In[2]:
+
+
+#-------------------------------------------------------------------------------------------------------------------------------
+# By Alexandra Lee (Septermber 2018) 
+#
+# Generate input files
+#
+# Dataset: Pseudomonas aeruginosa gene expression compendium referenced in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5069748/
+# 
+# Group samples based on gene expression of PA1673
+#
+# Generate offset vector using extreme gene expression values (train_offset_original):
+# average highest gene expression - average lowest gene expression 
+#-------------------------------------------------------------------------------------------------------------------------------
+import os
+import pandas as pd
+import numpy as np
+from scipy.stats import variation
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+randomState = 123
+from numpy.random import seed
+seed(randomState)
+
+
+# In[3]:
+
+
+# Load arguments
+data_file = os.path.join(os.path.dirname(os.getcwd()), "data", "all-pseudomonas-gene-normalized.zip")  # repo file is zipped
+gene_id = "PA1673"
+
+# Output training
+train_highest_file = os.path.join(os.path.dirname(os.getcwd()), "data", "PA1673_full_old", "train_highest_PA1673.txt")
+train_lowest_file = os.path.join(os.path.dirname(os.getcwd()), "data", "PA1673_full_old", "train_lowest_PA1673.txt")
+train_input_file = os.path.join(os.path.dirname(os.getcwd()), "data", "PA1673_full_old", "train_model_input.txt.xz")
+
+# Output test
+test_lowest_file = os.path.join(os.path.dirname(os.getcwd()), "data", "PA1673_full_old", "test_lowest_PA1673.txt")
+test_mid1_file = os.path.join(os.path.dirname(os.getcwd()), "data", "PA1673_full_old", "test_mid1_PA1673.txt")
+test_mid2_file = os.path.join(os.path.dirname(os.getcwd()), "data", "PA1673_full_old", "test_mid2_PA1673.txt")
+
+# Output offset
+original_offset_file = os.path.join(os.path.dirname(os.getcwd()), "data", "PA1673_full_old", "train_offset_original.txt")
+
+
+# In[4]:
+
+
+# Read in data
+data = pd.read_table(data_file, header = 0, sep = '\t', index_col = 0, compression='zip')
```

Comment:
no spaces in argument `=` in python pep8

Reply:
fixed in all files